EventFilter/Phase2PixelRawToDigi/plugins/PixelToRawProducer.cc

```diff
@@ -0,0 +1,257 @@
+// -*- C++ -*-
```

Comment:
All classes should have a comment at the top explaining what they are for (in the header file if they have one, but in the cc if they don't)